src/system/NewForm/FormAutocompleteMultiselect.stories.jsx

```diff
@@ -0,0 +1,107 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * Internal dependencies
+ */
+import { useState } from 'react';
+import * as Form from '.';
+
+export default {
+	title: 'Form/AutocompleteMulti',
+	argTypes: {
+		placeholder: {
+			type: { name: 'string', required: false },
+			control: { type: 'text' },
+		},
+		label: {
+			type: { name: 'string', required: false },
+			control: { type: 'text' },
+		},
+	},
+};
+
+const options = [
+	{ value: 'chocolate', label: 'Chocolate' },
+	{ value: 'strawberry', label: 'Strawberry Chocolate Vanilla Chocolate Vanilla' },
+	{ value: 'vanilla', label: 'Vanilla' },
+	{ value: 'pistachio', label: 'Pistachio' },
+	{ value: 'bubblegum', label: 'Bubblegum' },
+	{ value: 'ube', label: 'Ube' },
+	{ value: 'mango', label: 'Mango' },
+	{ value: 'buko', label: 'Buko' },
+	{ value: 'durian', label: 'Durian' },
+	{ value: 'lecheflan', label: 'Leche Flan' },
+	{
+		value: 'averylongicecreamflavor',
+		label: 'This is a very long ice cream flavorrrr with no name but it tastes goooooooood...',
+	},
+];
+
+const args = {
+	label: 'Ice Cream Flavors',
+	options,
+};
+
+// eslint-disable-next-line react/prop-types
+const DefaultComponent = ( { label = 'Label', width = 250, ...rest } ) => {
+	const [ selectedValue, setSelectedValue ] = useState( [] );
+
+	return (
+		<>
+			<Form.Root>
+				<div sx={ { width } }>
+					<Form.AutocompleteMulti
+						forLabel="form-autocompletemultiselect"
+						label={ label }
+						onChange={ obj => {
```

Comment:
Do we have a way to know what was the action when it was triggered? I can see it being useful to know if it was an addition or a deletion and which object was added/deleted

Reply:
Makes sense, I see the usefulness of this feature. Thank you for pointing this out! I'll try to add tracking of selectedOptions (increase/decrease).